tldw_Server_API/app/core/TTS/adapters/omnivoice_sidecar_server.py

```
@@ -0,0 +1,119 @@
+    @app.post("/control/shutdown", response_model=OmniVoiceHealthResponse)
+    async def shutdown(_: None = Depends(require_sidecar_token)) -> OmniVoiceHealthResponse:
+        return OmniVoiceHealthResponse(status="shutting-down", ready=False)
+
+    @app.post("/v1/synthesize")
+    async def synthesize(
+        request: OmniVoiceSynthesizeRequest,
+        _: None = Depends(require_sidecar_token),
+    ) -> Response:
+        if request.mode == "clone":
+            reference_path = Path(request.reference_audio_path or "")
+            if not reference_path.exists():
```

Ruff (0.15.10)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)